build/three.js

```diff
@@ -17042,6 +17043,24 @@ THREE.Mesh.prototype.getMorphTargetIndexByName = function ( name ) {
 
 };
 
+THREE.Mesh.prototype.makeVertexGetter = function () {
+
+	// we need this to avoid geometry type checks in actual getter
```

Comment:
How about adding `getVertex` directly to `Geometry` and `BufferGeometry` instead?